### ISSUE TITLE
[Mobile NeRF] Question: Training on a single GPU

### ISSUE THREAD

**Issue by DiegoHernandezKG** [google-research/jax3d#104]
Hello! First of all, congratulations on such an amazing paper and thank you very much for making the code public. I had a question regarding the requirements for training, is there a reason why one shouldn't attempt to train with a single GPU? It says that 8 V100 GPUs are required to "successfully train the model" However, I tested it with a single RTX3090 GPU and the results seem to be around the same as the paper (i.e. Same mesh quality as in the paper and around the same training times mentioned in the repository) 

![Screenshot from 2022-08-08 10-45-11](https://user-images.githubusercontent.com/82437256/183322555-70bdd0fb-3ad0-4ed9-a6c0-3a354f136162.png)

**Comment by tlightsky** [comment]
> 

I also try with `lego` model in single 3090 successfully

**Comment by zhenglin1991** [comment]
> > 
> 
> I also try with `lego` model in single 3090 successfully

Hi ,
would like to know how long it will take to finish the whole training process ?  other scenes would also train successfully ? I used the single RTX3080 and it failed because of OOM

**Comment by tlightsky** [comment]
> > > 
> > 
> > 
> > I also try with `lego` model in single 3090 successfully
> 
> Hi , would like to know how long it will take to finish the whole training process ? other scenes would also train successfully ? I used the single RTX3080 and it failed because of OOM

Almost same time(maybe a bit less) described in the paper,
that's the most strange part, cause I don't got 8 A100....

**Comment by tlightsky** [comment]
I try in WSL btw

**Comment by taiya** [comment]
We had a 8x GPU machine and always used that.
Yes, it's possible that for simpler scenes you can get by with not as many accelerators.

**Comment by tlightsky** [comment]
> We had a 8x GPU machine and always used that. Yes, it's possible that for simpler scenes you can get by with not as many accelerators.

Hi, is it possible to combine instant-ngp with mobile-nerf?

**Comment by czq142857** [comment]
Yes. It is just a bit harder to implement than the current MLP version.

**Comment by lishulincug** [comment]
> > > 
> > 
> > 
> > 我也尝试在单3090成功使用的型号`lego`
> 
> 嗨，想知道完成整个培训过程需要多长时间？其他场景也会成功训练？我使用了单个RTX3080，但由于OOM而失败

I used 6 v100 GPUs，but had the same problem：
![image](https://user-images.githubusercontent.com/13003282/194825922-9fe9870a-ea23-4e3f-83f5-86c1b83a9e09.png)

**Comment by lishulincug** [comment]
set  test_batch_size to smaller，Now it can run。
![image](https://user-images.githubusercontent.com/13003282/194838491-9f105fdd-d686-4180-90f5-edd62b6109f6.png)

**Comment by heidapao2b** [comment]
> Hello! First of all, congratulations on such an amazing paper and thank you very much for making the code public. I had a question regarding the requirements for training, is there a reason why one shouldn't attempt to train with a single GPU? It says that 8 V100 GPUs are required to "successfully train the model" However, I tested it with a single RTX3090 GPU and the results seem to be around the same as the paper (i.e. Same mesh quality as in the paper and around the same training times mentioned in the repository)
> 
> ![Screenshot from 2022-08-08 10-45-11](https://user-images.githubusercontent.com/82437256/183322555-70bdd0fb-3ad0-4ed9-a6c0-3a354f136162.png)

Can you tell me your requirement.txt
